modules/ocf_ldap/files/ocf.schema

```diff
@@ -76,10 +76,25 @@ attributetype ( 1.3.6.1.4.1.41759.2.2.3
         SUBSTR caseIgnoreSubstringsMatch
         SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 
+# based on ipHostNumber, 1.3.6.1.1.1.1.19
+attributetype ( 1.3.6.1.4.1.41759.2.2.4
+        NAME 'ip6HostNumber'
+        DESC 'IPv6 address, as represented by RFC1884 section 2.2.2'
+        SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+
+# based on ipHost, 1.3.6.1.1.1.2.6
+objectclass ( 1.3.6.1.4.1.41759.2.1.2
+        NAME 'ip6Host'
+        DESC 'Host supporting IPv6'
+        SUP top
+        AUXILIARY
+        MUST ( ip6HostNumber $ cn )
+        MAY ( manager $ description $ l ) )
```

Comment:
Why do you need a separate object class for IPv6 hosts? Is this just to enforce the IP is added?

I think we'd just rely on `ldap-lint` for that since we rely on it for a bunch of other checks already that are more complicated than we want to encode into ldap rules.

Reply:
There isn't a solid reason, but this matches the existing hierarchy of how we include `ipAddress`. It seemed weird to do `SUP ipHost` and then directly `MAY ip6address`.